surfactant/infoextractors/pe_file.py

```diff
@@ -233,6 +238,17 @@ def get_assemblyref_info(asmref_info):
     return asmref
 
 
+def get_implmap_info(im_info, imp_modules):
```

Comment:
```suggestion
def get_implmap_info(im_info, imp_modules):
```

This is doing a bit more than just getting the implmap -- maybe call it `add_implmap_info` or `insert_implmap_info` instead to capture that it is also modifying part of the SBOM.